Stripe/PublicHeaders/STPCard.h

```diff
@@ -110,6 +110,11 @@ typedef NS_ENUM(NSInteger, STPCardFundingType) {
 /**
  *  The cardholder's address.
  */
+@property(nonatomic, copy, nullable) STPAddress *address;
```

Comment:
We were getting a warning from the analyzer because the implementation of this can return nil even though the super class says nonnull, so I added this to correct it. If you think the implementation should be changed instead we could do that.